static/app/components/events/interfaces/keyValueList/index.tsx

```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```

Comment:
i think we're avoiding importing React this way

Reply:
I'll update to use a destructed import!